pkg/config/app/kuma-cp/config.go

```diff
@@ -204,9 +204,10 @@ var DefaultConfig = func() Config {
 		DpServer:    dp_server.DefaultDpServerConfig(),
 		Access:      access.DefaultAccessConfig(),
 		Experimental: ExperimentalConfig{
-			GatewayAPI:          false,
-			KubeOutboundsAsVIPs: true,
-			KDSDeltaEnabled:     false,
+			GatewayAPI:                      false,
+			KubeOutboundsAsVIPs:             true,
+			KDSDeltaEnabled:                 false,
+			UseTagFirstVirtualOutboundModel: true,
```

Comment:
this default to true is only temporary, I would like to run all e2e tests with this change on ci